index.html

```diff
@@ -14,6 +18,103 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="container">
+    <header>
+      <img class="headshot" src="Image\Head Shot.jpg" alt="Elan's Head Shot">
+      <div class="intro">
+        <nav>
+          <a href="#">Home</a>
+          <a href="#">about</a>
+          <a href="https://zealll.github.io/portfolio-website/">Portfolio</a>
+          <a href="#">about</a>
+        </nav>
+        <h1>Elan <span>Riznis</span></h1>
+        <address>
+          4310 44th Street, Sunnyside, NY 11104|(C) 917-373-5125|eriznis@gmail.com
+        </address>
+    </header>
+
+    <div>
+      <section>
+        <p>Customer-oriented, young and enthusiastic server with excellent interpersonal and communication skills.
+          Licensed in preparation and sale of alcoholic beverages, bar procedures and liquor laws.</p>
+      </section>
+    </div>
+    <div class="mid">
```

Comment:
```suggestion
    <div class="mid">
```
Would have liked to see this be a <section> rather than a <div> maybe.